customers-tomcat-legacy/README.md

```diff
@@ -0,0 +1,65 @@
+# Customer-tomcat-legacy application
+
+## Goals
+In this example, you will learn the insights on what changes are required to containerize a Java application running on tomcat server. This example uses Tackle operator to perform analysis on the application source code, the results of which produces insights needed to containerize the app.
+
+## Prerequisites 
+* Minikube
+* Podman
+* Tackle Operator
+
+## Tackle Operator Installation 
+To install tackle operator in your minikube cluster, follow along these set of [instructions](https://github.com/konveyor/tackle2-operator#tackle-operator-installation-on-k8s)
+
+_NOTE If you are using macOS, please follow along [this guide](https://github.com/konveyor/tackle2-operator/blob/main/docs/installation-macos.md) for tackle operator installation._
+
+## Get started with Analysis
+
+1. Create an app entry called `customer-tomcat` in the `Analysis` tab of the tackle ui. 
+    * Fill in the name of the application
+    * Expand source code and enter the following
+        * Repository Type: Git
+        * Source Repository: https://github.com/konveyor/example-applications
+        * Branch: main
+        * Root path: /customers-tomcat-legacy/
+        ![](https://i.imgur.com/PSF7g30.png)
+
+    * Click on the `Create`  button.
+
+2. Select the `customer-tomcat` app and click on the `Analyze` button.
+![](https://i.imgur.com/nT6mwzn.png)
+
+    * An application analysis wizard will open. 
+        * Select `Source Code` in the dropdown and click `Next` button
+        ![](https://i.imgur.com/YODEXqe.png)
+
+
+        * In the `Set Targets` pane, select `Containerization` option and click `Next`
+        ![](https://i.imgur.com/UXkGAaE.png)
+
+        * For scope, select `Application and internal dependencies only` 
+        ![](https://i.imgur.com/dHg0JRZ.png)
+
+        * Keep the defaults in Advanced section
+        * Move to the `Review` pane, and click on `Run` button
+        ![](https://i.imgur.com/IM8MY4C.png)
+
+
+3. You will notice that the status of the app has been updated to `In Progess`. Wait until it changes to `Completed`.
+    a.![](https://i.imgur.com/wM1j9hk.png)
```

Comment:
As a follow-up, can we add these images to the repo and link them that way? This will make it so that if Imgur ever does anything bad, we keep them.

Reply:
NVM it is a TODO sorry!